homeassistant/components/sensor/waqi.py

```diff
@@ -76,9 +75,9 @@ def __init__(self, data, station_id):
     def name(self):
         """Return the name of the sensor."""
         try:
-            return 'WAQI {}'.format(self._details['city']['name'])
+            return self._details['city']['name']
         except (KeyError, TypeError):
-            return 'WAQI {}'.format(self._station_id)
```

Comment:
I'd rather keep the "WAQI" text in front, especially in this instance where the name of the sensor will be just digits.